README.md

```
@@ -1,14 +1,31 @@
+[![Firebase Deploy](https://github.com/KoheiKanagu/blooms/actions/workflows/firebase_deploy.yml/badge.svg)](https://github.com/KoheiKanagu/blooms/actions/workflows/firebase_deploy.yml)
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)